Update README.md (#300)
w3f/Open-Grants-Program: README.md

  <img src="./src/Open_Grants_Program.png" style="width:1300px";>
</p>

> **:loudspeaker: Important:** The Web3 Foundation _Open Grants_ Program focuses on technology grants up to **USD 30k for initial grants** and **USD 100k for follow-up grants**, which are tracked **transparently** on GitHub and disbursed in **Bitcoin or DAI**.

Reviewer: I would leave the accentuation because it makes clear there are two programs. It's too easy to glance over the name of the program and not everyone might know.

Reviewer 2: agree

:earth_asia: **_This page is also available in [Chinese (中文)](./README_CN.md)._**

---

- [:wave: Introduction](#wave-introduction)

>   - [Requirements](#requirements)

Author: ```suggestion
  - [Guidelines](#guidelines)
```

## :wave: Introduction

> As part of our commitment to promoting the Web3 ecosystem, we offer comprehensive grants programs focused on funding software development and research efforts related to **Polkadot, Kusama and Substrate**. For more information about the Web3 Foundation please visit the [About page](https://web3.foundation/about/) of our website.

Author: ```suggestion
As part of our commitment to promoting the Web3 ecosystem, we offer comprehensive grants programs focused on funding software development and research efforts related to **Polkadot, Kusama and Substrate**. For more information about the Web3 Foundation please visit the [About page](https://web3.foundation/about/) on our website.
```

As part of our commitment to promoting the Web3 ecosystem, we offer comprehensive grants programs focused on funding software development and research efforts related to **Polkadot, Kusama and Substrate**. For more information about the Web3 Foundation please visit the [About page](https://web3.foundation/about/) of our website.

> ### :clipboard: About

Reviewer: You took this headline off the TOC. I have a plugin that generates TOCs automatically, so next time I commit something it'll be back in. If you don't like it in there, could you add ` <!-- omit in toc -->` after the heading?

- fully transparent,
- simple, and
- effective.

### Guidelines

Anyone is welcome to apply for a grant. Projects funded through our programs are broad in scope, but our focus lies on strong technical projects that add value to the ecosystem.

Generally, your project will have better chances to get accepted if:
- It presents a **well-researched** or tested concept, for which ideally you are able to show some prior work.
- You can demonstrate that the project will be **maintained** after completion of the grant, be it through an obvious commitment to the technology from your side, additional funding sources or an existing business model.
- Your team has **proven experience** with the relevant languages and technologies and/or a strong technical background. We ask you to provide the GitHub profiles of all team members as part of your application and will examine these for past activity and code quality. Naturally, you can also link to projects on other platforms.
- Your application is **rich in technical details** and well-defined.

> - You can clearly present how your project stands out among competitors or implements technology that doesn't exist in the ecosystem yet.

Reviewer: really good stuff

Additionally, it must fulfill the following requirements:

> - All code produced as part of a grant must be **open-sourced**, and it must also not rely on closed-source software for full funcionality. We prefer Apache 2.0 but the GNU GPL v3 license or Unlicense are also acceptable. 

Author: ```suggestion
- All code produced as part of a grant must be **open-sourced**, and it must also not rely on closed-source software for full funcionality. We prefer Apache 2.0, but GPLv3 or Unlicense are also acceptable.
```

- We do not award grants to teams that have successfully conducted a token sale or otherwise launched their own token unless there is a clear reason for it. If this applies to you, make sure to clearly present your case and explain why you cannot self-fund this project in your application.

> - Lastly, we do not provide funding for gambling-related projects.

Reviewer 2: Could we make this more general: We probably should also not fund stuff related to drugs, guns, etc.

Reviewer 2: see https://github.com/w3f/General-Grants-Program/pull/406

1. **Application:**
   1. [Fork](https://github.com/w3f/Open-Grants-Program/fork) this repository

>    2. In the newly created fork, create a **copy** of the application template ([`applications/application-template.md`](./applications/application-template.md)). If you're using the GitHub web interface, make sure you create a new file and copy the content of the template inside the new file. _Don't change the template file directly!_

Reviewer: > If you're using the GitHub web interface

Doesn't that apply to anyone?

Author: Nice catch.

```suggestion
   2. In the newly created fork, create a copy of the application template ([`applications/application-template.md`](./applications/application-template.md)). Make sure you create a new file and copy the [contents](https://raw.githubusercontent.com/w3f/Open-Grants-Program/master/applications/application-template.md) of the template inside the new one, and _do not modify the template file directly._
```

Author: It is true that using the web interface you can't just copy the file.
